nautobot/docs/models/extras/gitrepository.md

```diff
@@ -12,7 +12,10 @@ Some text-based content is more conveniently stored in a separate Git repository
 
 ## Repository Configuration
 
-When defining a Git repository for Nautobot to consume, the `name`, `remote URL`, and `branch` parameters are mandatory - the name acts as a unique identifier, and the remote URL and branch are needed for Nautobot to be able to locate and access the specified repository. Additionally, if the repository is private you may specify a `token` and the token `username` that can be used to grant access to the repository.
+When defining a Git repository for Nautobot to consume, the `name`, `remote URL`, and `branch` parameters are mandatory - the name acts as a unique identifier, and the remote URL and branch are needed for Nautobot to be able to locate and access the specified repository. Additionally, if the repository is private you may specify a `token` and any associated`username` that can be used to grant access to the repository.
+
+!!! important
```

Comment:
TBH I know this seems minor but I feel like this should be a warning vs. important since it is indicating that current approaches may be deprecated. Not a strong opinion tho.

Reply:
Your reasoning seems sound to me - will change.